internal/website/content/howto/blob/_index.md

```diff
@@ -9,3 +9,246 @@ providers and accessing them via HTTP. These guides show how to work with
 blobs in the Go CDK.
 
 <!--more-->
+
+## Opening a Bucket {#opening}
+
+The first step in interacting with unstructured storage is connecting to your
+storage provider. Every storage provider is a little different, but the Go CDK
+lets you interact with all of them using the [`*blob.Bucket` type][].
+
+[`*blob.Bucket` type]: https://godoc.org/gocloud.dev/blob#Bucket
+
+The easiest way to open a blob is using [`blob.OpenBucket`][] and a URL
+pointing to the blob, making sure you ["blank import"][] the driver package to
+link it in. See [Concepts: URLs][] for more details. If you need
+fine-grained control over the connection settings, you can call the constructor
+function in the driver package directly (like `s3blob.OpenBucket`). This guide
+shows how to use both forms for each provider.
+
+Once you have opened a bucket for the storage provider you want, you can
+store and access data from it using the standard Go I/O patterns.
+
+[`blob.OpenBucket`]:
+https://godoc.org/gocloud.dev/blob#OpenBucket
+["blank import"]: https://golang.org/doc/effective_go.html#blank_import
+[Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
+
+### Prefixed Buckets {#prefix}
```

Comment:
I think we can drop this section and just leave it in the godoc. Thoughts?

Reply:
I haven't studied godoc in detail, but at a first glance the content does not look identical, so it is not obvious to me what the equivalent would be. How about revisiting this separately? We can file an issue to reconsider this section.

Reply:
File another issue #2510